source/includes/api/facts/multiple-mongosync-endpoints.rst

```diff
@@ -0,0 +1,3 @@
+When you configure multiple ``mongosync``
+instances, you must send identical API endpoint commands to each
+``mongosync`` instance. 
```

Comment:
**Suggestion**

Can you add one more example on this page, which is basically Pause a ``mongosync`` Instance but with 2 instances, and then link to that example in this include? 

```suggestion
``mongosync`` instance. For an example of sending multiple commands, see :ref:`c2c-sharded-pause-multiple`.
```

Reply:
I'm not sure if another example is necessary because on the multiple mongosyncs page under each heading (for ex, "[Pause a mongosync Instance](https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-32690/multiple-mongosyncs/#pause-a-mongosync-instance)", it notes that the api commands affect sync on a single shard and that users need to use curl or another http client to repeat the api command on another instance.